.github/workflows/pr_diff.yaml

```
@@ -0,0 +1,48 @@
+name: PR Diff for Helm chart
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```

---

We can bar this off with [permissions](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs):
```suggestion
  pull_request_target:
  
permissions:
  pull-requests: write
```